README.md

```diff
@@ -183,19 +183,17 @@ This frontend of this project will be implemented with React and TypeScript and
 - You can disconnect from the SSH session. And once you log back in, you can resume your session by typing in `screen -r`. This is assuming that you have only one detached session.
 - Set up AWS API gateway(future devs don't need to do this again) to redirect incoming HTTPS request to our running server above. HTTPS is required because of the fact that the get current location functionality (from `navigator.geolocation.getCurrentPosition` API) only works under SSL.
 #### Frontend
-- TODO
+- The Frontend is now deployed on https://dubmap.herokuapp.com/ .
 ---
 
-### How To Run And Use DubMaps
+### How To Run And Use DubMap
 - Open Your browser
-- Visit the link https://yingfc.github.io/cse403 (this link is a placeholder)
+- Visit the link https://dubmap.herokuapp.com/
 - You can now use DubMaps and its functionalities
 ---
 
 ### Use Cases
 Ready for Beta Release (Feb 14th)
 - **Route navigation**: User can now enter course information (e.g. CSE 403 A) in the search bar and the map will show the navigation path from user's current location to the target building where the class is held.
 - **Room reservation**: User can now hover over to the building/library that are available for reservation, and a info window will show up with clickable link for the user to be redirected to the UW reservation system for room booking.
-
-Work in Progress
 - **Dining options**: User should be able to click a "show nearby dining options" button to see a list of currently opening restaurants around campus. And upon clicking the specific restaurant, it will show the navigation path to the dining/cafe on the map.
```

Comment:
Might better change this to say the last use case is **Ready for Final Release (March 7th)**